### PR TITLE
feat: Coverage new engine as data source for diff tabs ALA-789

### PR DESCRIPTION
**Pull request by nicklem**
Mentions the Codacy UI data source updates on the release note page for the new Coverage engine.

### :eyes: Live preview
https://ala-789-doc-update-the-coverage-rel--docs-codacy.netlify.app/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks/

### :construction: To do
- [x] Confirm this impacts only the [PR page diff tab](https://docs.codacy.com/repositories-coverage/pull-requests/#diff-tab).

### CODE REVIEW

File: docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md [hunk 1]
```diff
@@ -12,12 +12,38 @@ As part of an ongoing effort to improve the speed and value of the insights prov
...
+    <tr>
+      <td rowspan="2">Git providers</td>
+      <td><a href="#status-checks">Git provider status checks</a></td>
+      <td>Live (beta)</td>
+      <td>The new Coverage engine now sends coverage data to your Git provider.<br>This data is marked [beta] and is shown alongside the data from the current engine.</td>
+    </tr>
+    <tr>
+      <td><a href="#coverage-summaries">GitHub coverage summaries</a></td>
+      <td>Live</td>
+      <td>-</td>
```
Comment (nicklem) [inline docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md:32]:
```suggestion
      <td>The new Coverage engine now sends coverage summaries to GitHub. If you haven't done so yet, please review and update your app permissions as mentioned above.</td>
```

**Review by machadoit — APPROVED** [review]
LGTM!

**Comment by nicklem** [comment]
Ready to merge

File: docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md [hunk 2]
```diff
@@ -95,7 +124,23 @@ If you are using the old status check to block merging pull requests on GitHub,
...
+
+## Codacy app UI diff tabs from the new Coverage engine {: id="diff-tabs"}
+
+{%
+    include-markdown "../../assets/includes/coverage-github-accept-permissions.md"
+    start="<!--accept-permission-start-->"
+    end="<!--accept-permission-end-->"
+%}
+
+On January 1st 2024<!-- TODO ALA-789 date --> we set the [diff tab of the Coverage pull requests page](../../repositories-coverage/pull-requests.md#diff-tab) to get data from the new Coverage engine.
```
Comment (nicklem) [inline docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md:143]: Important: update with the actual launch date.

Comment (alerizzo) [inline docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md:143]:
```suggestion
On December 27th 2023<!-- TODO ALA-789 date --> we set the [diff tab of the Coverage pull requests page](../../repositories-coverage/pull-requests.md#diff-tab) to get data from the new Coverage engine.
```

Comment (claudiacarpinteiro) [reply docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md:143]:
```suggestion
On December 27th 2023 we set the [diff tab of the Coverage pull requests page](../../repositories-coverage/pull-requests.md#diff-tab) to get data from the new Coverage engine.
```

Comment (claudiacarpinteiro) [reply docs/release-notes/cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md]: Updated!

**Pull request merged**